packages/dds/sequence-sequential-id/README.md

```diff
@@ -0,0 +1 @@
+# @fluidframework/sequence-sequential-id
```

Comment:
Would be great to put at lease some description here :)

Reply:
Added description